### ISSUE TITLE
Velocity Cache - make debouncing optional

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#23595]
The new `$dotcache` tool is cool but it automatically "debounces" the cache.puts for 1 second.  This is good when a cache/page is under load but what this means in practice is that when doing something like the below, `$myVal` does not get set until 1 second after the initial call, which makes using the cached value problematic in some cases.
```
$dotcache.put("my-cache-key", $map)
#set($myVal = $dotcache.get("my-cache-key"))
$UtilMethods.isSet($myVal) == false

```

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/23598

**Comment by dsilvam** [comment]
Passed Internal QA. Tested with code similar than the one in the description.

**Comment by bryanboza** [comment]
Fixed, tested on release-23.01 // Docker // FF
